examples/tf/petting_zoo/atari/pong/feedforward/decentralised/run_mappo.py

```diff
@@ -0,0 +1,108 @@
+# python3
+# Copyright 2021 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Example running MAPPO on Cooperative Atari Pong."""
```

Comment:
I think it would be better to be explicit that this is cooperative pong in the file structure as well since most people would expect the default pong env

Reply:
Moved this to a different exp folder called coop pong.